test/test.dart

```diff
@@ -806,3 +807,18 @@ void printSplunkLogLink(AppNames appName, String? uriVariable) {
       "https://splunk.corp.mongodb.com/en-US/app/search/search?q=search index=baas$host \"${app.uniqueName}-*\" | reverse | top error msg&earliest=-7d&latest=now&display.general.type=visualizations");
   print("Splunk logs: $splunk");
 }
+
+List<SchemaObject> getSyncSchema() {
```

Comment:
Perhaps we can add a comment why we are doing this

Reply:
A comment is added.